tests/main.test.ts

```diff
@@ -1023,6 +1023,32 @@ describe('zip-it-and-ship-it', () => {
     },
   )
 
+  // TODO can be merged with the above once the FF is on `zisi_output_cjs_extension`
+  testMany(
```

Comment:
No need to use `testMany` if you're just testing a single bundler.

Reply:
Because this potentially will be merged again with the test above, and then be `testMany` again I would like to leave it as that.